system/HTTP/CURLRequest.php

```diff
@@ -548,17 +548,21 @@ protected function setCURLOptions(array $curlOptions = [], array $config = [])
 
         // SSL Verification
         if (isset($config['verify'])) {
-            if (is_string($config['verify'])) {
-                $file = realpath($config['ssl_key']) ?: $config['ssl_key'];
+            $configVerify = $config['verify'];
```

Comment:
Please don't introduce an additional variable, it makes the code less readable - use `$config['verify']`.

Reply:
Just to clarify. Array element $config['verify'] is accessed 8 times. Isn't this less efficient than using a local variable?

Reply:
> If the element of the array it's not changing value, you get a performance increase...

In this case, I'm pretty sure we would not see the difference. That's why I choose the readability.

> I forgot to ask. The breaking change is that $config['verify'] cannot take the value 'yes' anymore. This is the only breaking change?

Was using the value `yes` was documented somewhere? If not this can be skipped.
The breaking change is not using the `ssl_key` array key anymore.